src/main/resources/api/api.yaml

```diff
@@ -256,6 +309,33 @@ definitions:
   #
   # Quizzes
   #
+  V1Quiz:
+      required:
+        - id
+        - startedAt
+        - finishedAt
+        - testee
+      properties:
+        id:
+          type: integer
+          format: int64
+        startedAt:
+          type: string
+          format: date-time
+        finishedAt:
+          type: string
+          format: date-time
+        testee:
```

Comment:
hm... I think that it is a good idea to return here testee object.
please provide it also.

Reply:
@malinink Should I add description of testee object or it`s your issue?